spyql/parser.py

```diff
@@ -1,9 +1,11 @@
 from spyql.quotes_handler import QuotesHandler
-import spyql.utils
-import spyql.log
+from spyql import utils
+from spyql import log
 from spyql.processor import Processor
 from spyql.writer import Writer
 import re
+from spyql import agg
```

Comment:
🙃 I would place this line next to the other `spyql` imports, unless there is a specific reason (eg. PEP8 guideline) for it to be this way. Non blocking.